src/core/router/router.blueprint.tsx

```diff
@@ -280,7 +280,7 @@ export const blueprint: RouteObject[] = [
 ];
 
 function DefaultRoute() {
-  return <Navigate to="/credentials" />;
+  return <Navigate to="/verifications" />;
```

Comment:
why is the default route `verifications`?
because we don't have handle `credentials` yet?

Reply:
Yes